handbook/sales/index.md

```diff
@@ -114,3 +114,43 @@ Different customers and deals have different needs from our sales team. We segme
 - **Tier 1** = $100k+ ARR
 - **Tier 2** = $42k+ ARR
 - **Tier 3** = $22k+ ARR
+
+## Using HubSpot
+
+This page describes how we use HubSpot to maintain high data quality. This allows us to increase the effectiveness of all Sourcegraph teams through accurate insights. 
+
+Account Executives are responisble for maintaining HubSpot as a [source of truth](https://about.sourcegraph.com/handbook/communication#sources-of-truth). The data is synced between HubSpot and Looker every weekend so this data must be updated (at least) by end of day Friday. 
+
+### Associating Contacts to Deals
```

Comment:
In general, you should use sentence case (versus pascal case) per the style guide: https://about.sourcegraph.com/handbook/communication/style_guide